workflows/templates/storage.ceph-health-check.yaml

```diff
@@ -46,13 +46,16 @@ spec:
                 - name: scriptContent
                   value: |
                     sleep_time=15
-                    max_retries=45
+                    max_retries=60
                     attempt=1
                     while [[ $attempt -le $max_retries ]]; do
                       echo -e "\nChecking Ceph health status. Attempt ${attempt}/${max_retries}..."
                       if ! /opt/cray/tests/install/ncn/scripts/ceph-service-status.sh -v true; then
                         attempt=$(( attempt + 1 ))
                         sleep $sleep_time
+                        # Archive new crash events to avoid them affecting overall Ceph health status
+                        # Crashes can still be investigated with ceph crash ls/ceph crash info.
+                        ceph crash archive-all
```

Comment:
Sometimes Ceph performs self healing on certain cashes, but keeps crashes in history and health set to HEALTH_WARN. Archiving crashes only clears helath indicator, crashes still can be examined if needed.